app/src/main/java/com/pmatuki/wowpapers/view/detail/DetailViewModel.kt

```diff
@@ -16,6 +19,9 @@ internal class DetailViewModel : ViewModel() {
     @Inject
     lateinit var imageDownloadService: ImageDownloadService
 
+    @Inject
+    lateinit var wallpaperService: WallpaperService
+
```

Comment:
This injections should be added in the ViewModel constructor. If not, it will be very difficult to test or reuse this ViewModel if needed.

Reply:
Done!